java-shared-dependencies/dependency-analyzer/src/main/java/com/google/cloud/external/GitHubClient.java

```diff
@@ -0,0 +1,110 @@
+package com.google.cloud.external;
+
+import com.google.cloud.model.Interval;
+import com.google.cloud.model.PullRequest;
+import com.google.cloud.model.PullRequestStatistics;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * GitHubClient is a class that sends HTTP requests to the GitHub RESTful API.
+ * It provides methods for interacting with various GitHub resources such as repositories,
+ * issues, users, etc.
+ * <p>
```

Comment:
New line before `<p>`. No new line after `<p>`. https://google.github.io/styleguide/javaguide.html#s7.1.2-javadoc-paragraphs

Reply:
I formatted the code using
```
git diff -U0 origin/main | python google-java-format-diff.py -p1 -i
```